src/api/game/tinder/controller.js

```diff
@@ -52,6 +79,18 @@ export const fiberyImport = async ({ body: { fiberyPublicId }, user }, res, next
       runValidators: true,
     }).exec();
 
+    await Promise.all(
+      tinderGame.people.reduce(
+        (acc, { fiberyPublicId: personId }) => [
+          ...acc,
+          ...STATS_ACTIONS.map(action =>
+            Counter.ensureExists(generateStatsKey({ action, personId, slug: tinderGame.slug }))
```

Comment:
`acc.concat(Counter.ensureExists(generateStatsKey({ action, personId, slug: tinderGame.slug })))`

Reply:
suggested changes are not the same

However, `acc.concat(STATS_ACTIONS.map(...))` will probably do it

Reply:
Done.